TenancyInformationApi/Startup.cs

```diff
@@ -141,9 +140,12 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseHsts();
             }
 
-            //Get All ApiVersions,
-            var api = app.ApplicationServices.GetService<IApiVersionDescriptionProvider>();
-            _apiVersions = api.ApiVersionDescriptions.ToList();
+            if (app != null)
```

Comment:
When would `app` be null?

Reply:
I don't know why, but when I built from the command line, I got an error for not having a null-check.
When I just ran the build in Rider, it didn't care after I removed it.

Reply:
Everything is building fine on CircleCi without the null-check, so away it goes!